pyproject.toml

```diff
@@ -48,6 +48,7 @@ readme = {file = "README.rst", content-type = "text/x-rst"}
 [tool.setuptools_scm]
 write_to = "xesmf/_version.py"
 write_to_template = "__version__ = '{version}'"
+tag_regex = "^(?P<prefix>v)?(?P<version>[^\\+]+)(?P<suffix>.*)?$"
```

Comment:
This is the most common template used by many projects. When releasing we need to create a tag with the `v` like `v2.0`. If that is not desirable I can change it but the `v` helps find the tags b/c they all will start with the same letter.

Reply:
All good.